lib/restforce/concerns/api.rb

```diff
@@ -171,19 +171,27 @@ def org_id
 
       # Public: Executs a SOQL query and returns the result.
       #
-      # soql - A SOQL expression.
+      # soql    - A SOQL expression.
+      # options - An optional hash containing options for this
+      #           query.
       #
       # Examples
       #
       #   # Find the names of all Accounts
       #   client.query('select Name from Account').map(&:Name)
       #   # => ['Foo Bar Inc.', 'Whizbang Corp']
       #
+      #   client.query('select Name from Account', batch_size: 1).map(&:Name)
+      #   # => ['Foo Bar Inc.']
+      #
       # Returns a Restforce::Collection if Restforce.configuration.mashify is true.
       # Returns an Array of Hash for each record in the result if
       # Restforce.configuration.mashify is false.
-      def query(soql)
-        response = api_get 'query', q: soql
+      def query(soql, options = {})
```

Comment:
Doesn't this options conflict the variable defined here? https://github.com/restforce/restforce/lib/restforce/concerns/base.rb#L64

Reply:
It sure does. It's shadowing that. I'll change the name.

Reply:
Thanks, LGTM